### DISCUSSION TITLE
Rimuoverei tutte le faccende su privacy, cookie, analytics

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/ods-mkdocs-material#42] [Idee/Proposte]
Buongiorno,
sono questioni legali e le lascerei a chi vuole pubblicare contenuti e creare un nuovo sito.

Possiamo dare indicazioni su come si imposta l'analytics, ma non abiliterei nulla nel sito di default che abilitiamo in questo sito.

Che ne pensate?

**Comment by gbvitrano** [comment]
Si, attualmente il file privacy.md è collegato nel mkdocs.yml ma non è collegato lato utente, si può eliminare del tutto, ogni utente se vuole scrive il proprio, stessa cosa per i cookies.
Per analytics ok,  mettiamo solo le indicazioni, se non ricordo male non è inserito  nel  mkdocs.yml
